extension/src/popup/components/account/MobileAppBanner/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from "react";
+import { Icon, Text } from "@stellar/design-system";
+import { useTranslation } from "react-i18next";
+import browser from "webextension-polyfill";
+
+import {
+  getMobileAppBannerDismissed,
+  dismissMobileAppBanner,
+} from "@shared/api/internal";
+
+import { openTab } from "popup/helpers/navigate";
+import FreighterLogo from "popup/assets/logo-freighter-shadow.png";
+
+import "./styles.scss";
+
+// Check if browser storage is available (extension context, not fullscreen)
+const isStorageAvailable = (): boolean => {
```

Comment:
I think this check is a holdover from the last version of this PR, right? You shouldn't need to check this anymore. Whether you're in extension mode or in fullscreen mode, you should have access to the background storage now.

You should be able to safely remove this